test/zzz-correct-index-names.sh

```diff
@@ -43,7 +43,7 @@ cleanup() {
   oc logs $fpod > $ARTIFACT_DIR/$fpod.log 2>&1
 
   oc exec $fpod -- env | sort > $ARTIFACT_DIR/env_vars.log 2>&1
-  oc exec $fpod -- find /etc/fluent/configs.d/ -exec cat {} \; $ARTIFACT_DIR/fluent_conf.log 2>&1
+  oc exec $fpod -- find -type f /etc/fluent/configs.d/ -exec cat {} \; $ARTIFACT_DIR/fluent_conf.log 2>&1
```

Comment:
find /etc/fluent/configs.d -type f
the directory names must come first (at least, in old versions of find)